lib/experimental/class-gutenberg-rest-template-revision-count.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Template_Revision_Count class
+ *
+ * @package    gutenberg
+ */
+
+/**
+ * Gutenberg_REST_Template_Revision_Count class
+ *
+ * Template revision changes are waiting on a core change to be merged.
+ * See: https://github.com/WordPress/gutenberg/pull/45215#issuecomment-1592704026
+ * When merging into core, prepare_revision_links() should be merged with
+ * WP_REST_Templates_Controller::prepare_links().
+ */
+class Gutenberg_REST_Template_Revision_Count extends Gutenberg_REST_Templates_Controller_6_3 {
```

Comment:
I kept the naming convention from https://github.com/WordPress/gutenberg/pull/45215

Was considering `Gutenberg_REST_Templates_Controller`, but there are no other features here. Not sure.